src/test/java/lotto/domain/WinningLottoTest.java

```diff
@@ -40,6 +39,12 @@
     @Test
     void match() {
         WinningLotto winningLotto = WinningLotto.of("1,2,3,4,5,6");
-        assertThat(winningLotto.matchLottos(Arrays.asList(Lotto.of("1,2,3,4,5,6")))).contains(WinningLottoAmount.FIRST);
+        assertThat(winningLotto.matchLottos(Arrays.asList(Lotto.ofWinningLotto("1,2,3,4,5,6")))).contains(WinningLottoAmount.FIRST);
+    }
+
+    @Test
+    void 로또_2등_테스트() {
+        WinningLotto winningLotto = WinningLotto.createWithBonusNo("1,2,3,4,5,6", 7);
```

Comment:
> 구현을 하다보니 테스틀 위한 생성자와 메서드가 생기는데 이런 부분은 어떻게 해결할까요..?
(WinningLotto.class -> createWithBonusNo 같은 메서드가 생깁니다.)

WinningLotto의 생성자로 처리할 순 없나요?

```suggestion
        WinningLotto winningLotto = new WinningLotto("1,2,3,4,5,6", 7);
```